LEGO1/legoworld.h

```diff
@@ -39,9 +41,12 @@ class LegoWorld : public LegoEntity {
 	void EndAction(MxPresenter* p_presenter);
 
 protected:
-	undefined m_unk68[0x30];
-	LegoCameraController* m_camera;
-	undefined m_unk9c[0x5a];
+	LegoPathControllerList m_list0x68; // +0x68
```

Comment:
We haven't formalized the rule yet but so far we have not put a plus sign in front of the member offset